db/seeds.rb

```diff
@@ -0,0 +1,45 @@
+require 'csv'
+
+
+WORK_FILE = Rails.root.join('db', 'works-seeds.csv')
+puts "Loading raw work data from #{WORK_FILE}"
+
+work_failures = []
+CSV.foreach(WORK_FILE, :headers => true) do |row|
+  work = Work.new
+  work.category = row['category']
+  work.title = row['title']
+  work.creator = row['creator']
+  work.publication_year = row['publication_year']
+  work.description = row['description']
```

Comment:
I think you should be able to just pass the `row` hash into `Work.new`:

```suggestion
  work = Work.new(row)
```